lib/lib_install.php

```diff
@@ -41,14 +41,14 @@ function checkRequirements(string $dbType = ''): array {
 	$json = function_exists('json_encode');
 	$mbstring = extension_loaded('mbstring');
 	// @phpstan-ignore-next-line
-	$data = DATA_PATH && touch(DATA_PATH . '/index.html');	// is_writable() is not reliable for a folder on NFS
+	$data = DATA_PATH && file_exists(DATA_PATH . '/index.html') && touch(DATA_PATH . '/index.html');	// is_writable() is not reliable for a folder on NFS
 	// @phpstan-ignore-next-line
-	$cache = CACHE_PATH && touch(CACHE_PATH . '/index.html');
+	$cache = CACHE_PATH && file_exists(CACHE_PATH . '/index.html') && touch(CACHE_PATH . '/index.html');
```

Comment:
Please remove the `file_exist` as it might not exist

Reply:
I do not understand it

Reply:
`./data/` can for instance be mounted as an empty Docker volume. We do have a script initialising some minimal structure, but we should have as little requirements about existing files there as possible.
Plus, it does not provide any additional information.

Reply:
Please edit this PR as you wish. It would support me and this PR a lot. The discussion is very long around the `touch()`. I am not so familiar with this topics in details.

Reply:
No need to change anything to those lines. The whole `touch()` part was exactly as it should. `@` is fine, `is_writable()` does not work for instance on a folder on NFS, `fileperms()` is a bad idea as it is poorly multi-platform.